packages/veraswap-app/src/atoms/stargate.ts

```diff
@@ -46,15 +50,16 @@ export const stargateQuoteAtom = atomWithQuery((get) => {
 
     const bridgeSymbol = bridge.currencyIn.symbol!;
 
-    // TODO: handle USDC bridging
     if (bridge.currencyIn.isNative) {
         if (bridgeSymbol !== "ETH" || !(chainIn.id in STARGATE_POOL_NATIVE && chainOut.id in STARGATE_POOL_NATIVE)) {
             return disabledQueryOptions as any;
         }
-        // } else if (bridgeSymbol === "USDC") {
-        //     if (!(chainIn.id in STARGATE_POOL_USDC && chainOut.id in STARGATE_POOL_USDC)) {
-        //         return disabledQueryOptions as any;
-        //     }
+    } else if (bridgeSymbol in STARGATE_TOKEN_POOLS) {
+        const pools = STARGATE_TOKEN_POOLS[bridgeSymbol as keyof typeof STARGATE_TOKEN_POOLS];
```

Comment:
is the `as` needed if the `if`statement is already `bridgeSymbol in STARGATE_TOKEN_POOLS`?